lib/cadence/workflow/decision_task_processor.rb

```diff
@@ -8,14 +8,15 @@ class Workflow
     class DecisionTaskProcessor
       MAX_FAILED_ATTEMPTS = 50
 
-      def initialize(task, domain, workflow_lookup, client, middleware_chain)
+      def initialize(task, domain, workflow_lookup, connection, middleware_chain, config)
```

Comment:
Would love to see some comments in at least some of these classes, like what's a connection, what's a config, or how do I get them to pass in?

Reply:
Yes, point taken, will think of a solution 👍